### ISSUE TITLE
Inference from custom model 

### ISSUE THREAD

**Issue by koopa31** [instanseg/instanseg#45]
Hi,
I trained my own model and have a folder called my_first_instanseg. For now, I don't wanna publish it on Biomodel zoo, but I wanna use it on my images. I don't understand how to use inference.py. I tried to use the export_model notebook and got a .pt file. From what I understand, inference needs a bioimage zoo model, or a nn.Module, or a local bioimage format model. Currently, my folder contains :

- epoch_outputs
- experiment_log.csv
- experiment_metrics.csv
- f1_metric.png
- loss.png
- model_weights.pth

Is there an easy way to just instantiate the model and load the weights to make predictions? I saw an issue was opened concerning the creation of a class similar to the Cellpose one to facilitate its use.

Thanks!

**Comment by ThibautGoldsborough** [comment]
from instanseg import InstanSeg
my_instanseg = InstanSeg("my_first_instanseg") 

(That's if you ran the export_model.ipynb)

**Comment by ThibautGoldsborough** [comment]
An alternative way:

`torchscript_object = torch.jit.load("path_to_your_pt_file.pt")

from instanseg import InstanSeg
my_instanseg = InstanSeg(torchscript_object)`
